custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
```

Comment:
Consider updating the text content to make it clearer and more concise. Maybe include specific details or achievements. [View Reference](https://github.com/raiaqt/betterai-portfolio)